var/spack/repos/builtin/packages/r/package.py

```diff
@@ -19,7 +19,7 @@ class R(AutotoolsPackage):
     url = "https://cloud.r-project.org/src/base/R-3/R-3.4.3.tar.gz"
 
     extendable = True
-
```

Comment:
Please preserve the vertical space between the variable setting and the version specifications.